pkg/graph/schema.graphql

```diff
@@ -2710,3 +2711,23 @@ enum SystemIntakeMeetingState{
   SCHEDULED
   NOT_SCHEDULED
 }
+
+"""
+This represents the statuses that and admin would see as a representation of a system intake
```

Comment:
It might be helpful to add a note that there's no value for the status of a new, ready-to-start request (as shown in [this Figma](https://www.figma.com/file/ChzAP34A2DVvQUNQwD7lCt/IT-Governance-Next?type=design&node-id=475-109186&mode=design&t=XYVReFpvRphTZJKB-0)) because it's not visible to admins until the initial request form is in progress.